deploy/terraform-gcp/main.tf

```diff
@@ -0,0 +1,168 @@
+provider "google" {
+  project = "excellent-ion-277200"
```

Comment:
I believe the project will have to be filled in by the user or set by an env for their specific project in gcp.

Reply:
Thanks, that's a leftover from my test setup. I'll fix it.